adal/src/main/java/com/microsoft/aad/adal/SSOStateSerializer.java

```diff
@@ -66,7 +66,7 @@
      * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
      */
     @SuppressWarnings("unused")
-    private SSOStateSerializer() {
+    SSOStateSerializer() {
```

Comment:
All I did was make this `package-private` instead of `private`. The empty constructors are used in the unit tests. I just make them all the same for consistency.